libc/test/src/fenv/rounding_mode_test.cpp

```diff
@@ -26,19 +31,66 @@ TEST_F(LlvmLibcRoundingModeTest, SetAndGet) {
   EXPECT_EQ(s, 0);
   int rm = LIBC_NAMESPACE::fegetround();
   EXPECT_EQ(rm, FE_TONEAREST);
+  EXPECT_TRUE(LIBC_NAMESPACE::fputil::fenv_is_round_to_nearest());
```

Comment:
Test this last. Also should actually perform an FP operation to make sure it used the right mode